_content/articles/temple-where-tattoos-come-to-life_lastrucci-francesco.md

```diff
@@ -0,0 +1,21 @@
+---
+title: "Inside the Thai Temple Where Tattoos Come to Life"
+authors:
+  - "Francesco Lastrucci"
+external_url: ""https://www.nytimes.com/2021/10/25/travel/sak-yant-tattoo-thailand.html
```

Comment:
Urls go inside the quotes 😉 

Let's make the nytimes link the source_url and here do `external_url: "https://archive.is/eST6U"`